data.py

```diff
@@ -29,19 +29,42 @@ def __call__(self, samples):
 
         return all_texts, texts_mask, all_strokes, strokes_mask
```

Comment:
This is not from this review but from a previous commit that I might have missed. Why do we have a class for this? I would prefer to have a function rather than a class here. If it's for keeping the `device` there are other ways of doing that.

Reply:
What would be other ways of dealing with the `device`?

Reply:
Discussed in person about this yesterday. However, if we move to accelerate, it won't be useful anymore.